action/steps/activate.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-if [[ -n "$UNITY_LICENSE" ]]; then
+if [[ -n "$UNITY_LICENSE" || -n "$UNITY_LICENSE_PATH" ]]; then
```

Comment:
Or operator should be outside of the double square brackets for compatibility reasons

`if [[ -n "$UNITY_LICENSE" ]] || [[ -n "$UNITY_LICENSE_PATH"  ]]; then`

Reply:
![image](https://user-images.githubusercontent.com/1288575/75902266-b8f05280-5e7a-11ea-9b79-c3b32c25047a.png)

https://linuxacademy.com/blog/linux/conditions-in-bash-scripting-if-statements/

The syntax I see from here.

Reply:
I understand that works, but in my experience it's generally better to not have the or part of the condition at all.

There's a good plugin for modern code editors called `shellcheck` that will help you validate your script if you're afraid to make changes.